README.md

```diff
@@ -45,35 +48,35 @@ cd ovsrpro
 git checkout <tag>		# where tag is, for example 17.02.1
 mkdir ovsrpro-build
 cd ovsrpro-build
-cmake ../ovsrpro -DXP_STEP=build
+cmake ../ovsrpro -DXP\_STEP=build
```

Comment:
I'm not sure you want to add the slash here since it's already marked as a code snippet...